### PR TITLE
Email pw warning STABLE

### PR DESCRIPTION
**Pull request by dkmullen**
Ticket: #67359
This PR replaces #1978
Adds a message to make it clear that password (and matching PW) are always needed to submit the form (if SMTP auth is selected), even when you are just editing.

Incorporates requested change: T('Matching passwords must be entered to submit the form, even when editing settings.')

### CODE REVIEW

**Comment by dkmullen** [comment]
rebuild

**Comment by erinix** [comment]
looks okay to me, pending docs review

File: src/app/helptext/system/email.ts [hunk 1]
```diff
@@ -55,6 +55,10 @@ export const helptext_system_email = {
     validation: [Validators.required]
   },
 
+  em_pwmessage: {
+    paraText: T('Matching passwords must be entered to submit the form, even when editing settings.')
```
Comment (Mrt134) [inline src/app/helptext/system/email.ts]:
```suggestion
    paraText: T('Confirm the password to submit settings changes.')
```

Comment (Mrt134) [reply src/app/helptext/system/email.ts]: Simplification suggestion.

**Pull request merged**